main.gms

```diff
@@ -126,6 +126,8 @@ $elseif.loadData %DevMode% == 1 $call "RScript ./loadMadratData.R DevMode=1"
 $elseif.loadData %DevMode% == 2 $call "RScript ./loadMadratData.R DevMode=2"
 $endif.loadData
 $endif.genInp
+file fStat /'modelstat.txt'/; 
```

Comment:
Add `modelstat.txt` to `.gitignore` so Git ignores it when created in the root folder.

Reply:
Yes, the issue has been resolved.

Reply:
.gitignore  updated